docs/framework/github_feedback.md

```diff
@@ -0,0 +1,175 @@
+
+1. Release product
+2. Users use product, and have feedback: bug reports, specific feature requests, or general emotions about using
+3. They submit this feedback to you, the owner of the software product.
+4. You triage the feedback, looking for common trends and deciding if the feedback is worth implementing.
+5. You implement the pieces of feedback that you decided to implement.
+6. Repeat back to 1.
+
+This tool helps you solve steps 3 and 4; users submitting feedback to you, and you being able to triage that feedback.
+Since much docassemble development [already involves GitHub](../github.md), the AssemblyLine uses [GitHub's issues](https://docs.github.com/en/issues) to handle and triage feedback.
+GitHub issues let you tag user feedback as such, link issues to PRs that fix the code, and track
+when the fix has been added to the code.
+
+GitHub is more developer focused, but it's a good tool to use if you don't currently track interview issues. The [Feedback package](https://github.com/SuffolkLITLab/docassemble-GithubFeedbackForm) started off focused on the GitHub workflow, but has since evolved to handle manual workflows that are less dependant on GitHub. If your organization has a different software system used to triage feedback, you'll have to write your own code to automatically integrate with it.
+
+## How to use Feedback in an Assembly Line interview
+
+This how-to-guide will cover the simplest way to get feedback from an interview. By the end you'll have:
+
+* a link to give feedback in the footer of your interviews
+* a separate interview that will show either a tester feedback form or a general public feedback form
+* will save that feedback locally, and if desired, will make a GitHub issue with the information
+
+It won't:
+
+* tell you how to customize how you ask for feedback or what the interview looks like
+* integrate with other systems besides the current docassemble server and GitHub
+
+Before you start, we'll assume that you:
+
+* have access to an [admin account on a docassemble server](https://suffolklitlab.org/legal-tech-class/docs/classes/assembly-line/2020-assembly-line-assignment-1#before-you-get-started)
+* have the [`GithubFeedbackForm` package installed on your server](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/installation#run-the-installation-script)
+* know how to [edit the docassemble configuration (config) file](https://docassemble.org/docs/config.html#edit)
+* know [what the playground is](https://suffolklitlab.org/legal-tech-class/docs/classes/docacon-2020/hello-world#introduction-to-the-docassemble-playground) and [how to use it to develop a docassemble interview](https://suffolklitlab.org/legal-tech-class/docs/classes/docacon-2020/hello-world#hello-world)
+* know [what "blocks" are](https://suffolklitlab.org/legal-tech-class/docs/yaml#documents) in docassemble
+
+### Configuration
+
+In your docassemble Configuration, you'll add `github issues` attribute, like the below:
```

Comment:
Good additiion, I'll add it.

I wrote it like this because TBH, I'm not sure if that's the way we want to take the package fully? It still just sorta feels like a feature I'm adding on, while most people using it (just us and CaroRob) were leaning towards private GitHub repos instead of  storing on the server.